content/en/docs/release-oversight/disruption-tests.md

```diff
@@ -0,0 +1,213 @@
+---
+title: "Disruption Tests"
+description: An look at how disruption tests are architected and implemented.
+---
+
+## Summary
+
+Disruption tests are a set of tests that measure an endpoints resilience or an allowed alert duration. As the `e2e tests` are run the results are matched against historical data to determine if the disruption intervals are within expectations.
+
+### Resources
+
+> ⚠️ Note! You'll need access to the appropriate groups to work with disruption data, please reach out to the TRT team for access.
+
+- [Periodic Jobs](https://github.com/openshift/release/tree/master/ci-operator/jobs/openshift/release)
+- [BigQuery](https://console.cloud.google.com/bigquery?project=openshift-ci-data-analysis)
+- [DCPR Job Aggregation Configs](https://github.com/openshift/continuous-release-jobs/tree/master/config/clusters/dpcr/services/dpcr-ci-job-aggregation)
+- [Origin Synthetic Backend Tests](https://github.com/openshift/origin/tree/master/pkg/synthetictests/allowedbackenddisruption)
+
+## Architecture
```

Comment:
Please mention that this is the architecture diagram for how disruption data is stored on gcp/BigQuery and how query_results.json is generated and not the architecture for the Disruption Testing in general.

This one drawing really makes clear what's happening (and now I can explaining it a lot better) -- thx.

Reply:
Good point, updated.